doc/dev/tests.md

```diff
@@ -196,19 +206,19 @@ be found in the [package's README][python-dotenv_readme].
 
 If using a `New-TestResources` script from [/eng/common/TestResources][test_resources], the script should output any
 environment variables necessary to run live tests for the service. After storing these variables in your `.env` file
--- formatted as `VARIABLE=value` on separate lines -- your credentials and test configuration variables will be set in 
+-- formatted as `VARIABLE=value` on separate lines -- your credentials and test configuration variables will be set in
 our environment when running tests.
 
 If your service doesn't have a `test-resources` file for test deployment, you'll need to set environment variables
 for `AZURE_SUBSCRIPTION_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, and `AZURE_CLIENT_SECRET` at minimum.
 
 1. Set the `AZURE_SUBSCRIPTION_ID` variable to your organization's subscription ID. You can find it in the "Overview"
-section of the "Subscriptions" blade in the [Azure Portal][azure_portal].
+   section of the "Subscriptions" blade in the [Azure Portal][azure_portal].
 2. Define the `AZURE_TENANT_ID`, `AZURE_CLIENT_ID`, and `AZURE_CLIENT_SECRET` of a test service principal. If you do not
-have a service principal, use the Azure CLI's [az ad sp create-for-rbac][azure_cli_service_principal] command (ideally,
-using your alias as the service principal's name prefix):
-```
-az login
+   have a service principal, use the Azure CLI's [az ad sp create-for-rbac][azure_cli_service_principal] command (ideally,
+   using your alias as the service principal's name prefix):
+
+```az login
```

Comment:
This change causes the `az login` to be removed from the example:

![image](https://github.com/Azure/azure-sdk-for-python/assets/31998003/84b10d0a-0407-4d0a-aa3b-4c8a42d33322)